.github/workflows/rubocop.yml

```
@@ -0,0 +1,16 @@
+name: RuboCop
+on: [pull_request]
+env:
+  BUNDLE_WITHOUT: assets:console:development:dynflow_sidekiq:ec2:gce:journald:jsonp:libvirt:openid:openstack:ovirt:redis:service:telemetry:test:vmware
+jobs:
+  rubocop:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.6
```

2.6 is an odd version to test with. Currently all production runs on 2.5 but the next target will be 2.7. I don't think we'll ever ship with 2.6.
```suggestion
          ruby-version: 2.7
```

---

Ah, thanks for the info, I'll go for 2.5 then, as that's the TargetVersion for the Cops.